openmetadata-ui/src/main/resources/ui/src/components/Tag/TagsV1/TagsV1.component.tsx

```diff
@@ -90,7 +93,7 @@ const TagsV1 = ({
     () =>
       (tagType ?? tag.source) === TagSource.Glossary
         ? getGlossaryPath(tag.tagFQN)
-        : getTagPath(Fqn.split(tag.tagFQN)[0]),
+        : getClassificationTagPath(Fqn.split(tag.tagFQN).join('.')),
```

Comment:
We are splitting and joining again, Can you explain use case here?

Reply:
Updated.